pkg/util/proxyutil/reverse_proxy_test.go

```diff
@@ -101,6 +102,7 @@ func TestReverseProxy(t *testing.T) {
 			{
 				desc:               "Timed out request should return 504 Gateway timeout",
 				transport:          timedOutTransport,
+				responseWaitTime:   500 * time.Millisecond,
```

Comment:
Will we actually wait 500 milliseconds here? It seems a bit excessive, 100x should do and would slow down the tests less.

Reply:
Good point. Fixed